src/component/Header.module.css

```diff
@@ -0,0 +1,22 @@
+.header {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+}
+
+.header h2 {
+  font-size: 3rem;
+}
+
+.header span {
+  text-decoration: none;
+  color: inherit;
+  padding: 1.5rem;
+  margin-top: 1rem;
+  margin-right: 1rem;
+  font-size: 1.4rem;
+  transition-duration: 120ms;
+  transition-property: border, background-color;
+  border-bottom: 2px solid transparent;
+}
```

Comment:
- [ ] Kindly create a directory in your `./src` called `css-modules `that will store your css-modules. It is good practice to house all similar files in the same directory and import them where required. That way your project will be very easy to browse through. In addition, it will make it look more professional and stand out.